include/IImage.h

```diff
@@ -329,6 +329,12 @@ class IImage : public virtual IReferenceCounted
 	//! Sets a pixel
 	virtual void setPixel(u32 x, u32 y, const SColor &color, bool blend = false ) = 0;
 
+	//! Copies this surface into another, if it fits.

```

Comment:
The comment states “if it fits” but the implementation is “if it has the exact same size and format”